tree/tree/inc/TBasket.h

```diff
@@ -61,6 +63,21 @@ class TBasket : public TKey {
 
 public:
 
+   // The IO bits flag is to provide improved forward-compatibility detection.
+   // Any new non-forward compatibility flags related serialization should be
+   // added here.  When a new flag is added, set it in the kSupported field;
+   //
+   // If (fIOBits & ~kSupported) is non-zero -- i.e., an unknown IO flag is set
+   // in the fIOBits -- then the zombie flag will be set for this object.
+   //
+   enum EIOBits {
+      // The following to bits are reserved for now; when supported, set
+      // kSupported = kGenerateOffsetMap | kBasketClassMap
+      // kGenerateOffsetMap = BIT(1),
+      // kBasketClassMap = BIT(2),
+      kSupported = 0
```

Comment:
I'm not sure it's a foregone conclusion that `kSupported` should be `|` of all EIOBits.  Particularly, if there is an experimental feature that we decide to not put into production, we want the ability to define a IOBit but not support it.

Now, in a future PR when I add an interface in the `ROOT::Experimental` namespace to enable / set IOBits in a basket, we must include a check that the user-requested IOBit is supported.

Reply:
> Particularly, if there is an experimental feature that we decide to not put into production, we want the ability to define a IOBit but not support it.

This can be handled by adding a EObsoleteIOBits.  This has the additional great advantage of documenting for the user too the status of such bits (defined but not supported).

Reply:
Done!